easybuild/easyconfigs/m/MCR/MCR-R2013b.eb

```diff
@@ -0,0 +1,18 @@
+name = 'MCR'
+version = 'R2013b'
+
+homepage = 'http://www.mathworks.com/products/compiler/mcr/'
+description = """The MATLAB Runtime is a standalone set of shared libraries
+ that enables the execution of compiled MATLAB applications
+ or components on computers that do not have MATLAB installed."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [
+    'http://www.mathworks.com/supportfiles/MCR_Runtime/%(version)s/',
```

Comment:
Doesn't seem to work anymore, this may mean the `source_urls` in the existing `MCR` easyconfigs also need to be updated.

@migueldiascosta Did you found these to be broken in your `source_urls` checking? I don't see it listed in the gist linked from https://github.com/hpcugent/easybuild-framework/issues/2087?